Branch-SDK/src/main/java/io/branch/referral/BranchAsyncTask.java

```diff
@@ -16,14 +16,11 @@
      * @param params Params for executing this Async task
      * @return This object for method chaining
      */
-    public AsyncTask<Params, Progress, Result> executeTask(Params... params) {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
```

Comment:
min android api version set on the sdk ensures that this is statement is always true